_extras/design.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: page
+title: "Lesson Design"
+permalink: /design/
+---
+## Stage 0 - Assumptions
+
+*   Audience
+    *   Graduate students and practicing scientists 
+        *   Materials not specific to discipline
```

Comment:
But examples have to be drawn from somewhere: will you use a small data set as a running example, or jump around between data sets?